### PR TITLE
retroarch: Add version 1.10.0

### PR DESCRIPTION
**Pull request by beyondmeat**

### CODE REVIEW

File: bucket/retroarch-nightly.json [hunk 1]
```diff
@@ -0,0 +1,69 @@
+{
+    "homepage": "https://www.retroarch.com/",
+    "description": "Frontend for emulators, game engines and media players. (nightly)",
```
Comment (Calinou) [inline bucket/retroarch-nightly.json:3]:
```suggestion
    "description": "Frontend for emulators, game engines and media players (nightly)",
```

File: bucket/retroarch.json [hunk 2]
```diff
@@ -0,0 +1,70 @@
+{
+    "homepage": "https://www.retroarch.com/",
+    "description": "Frontend for emulators, game engines and media players.",
```
Comment (Calinou) [inline bucket/retroarch.json:3]:
```suggestion
    "description": "Frontend for emulators, game engines and media players",
```

**Review by Calinou — APPROVED** [review]
Thanks!

**Comment by rashil2000** [comment]
Hi @beyondmeat and @Calinou!

retroarch is already present in the Extras bucket. It's been present since ~4 years now and has various fixes and a more involved pre_install script.

I would recommend removing it from here (i.e. the Games bucket) to prevent duplication among the known official buckets.

**Comment by Calinou** [comment]
@rashil2000 I wasn't aware about that. I removed `retroarch` in https://github.com/Calinou/scoop-games/commit/a17ea5680af3db4191660b92704c19a6a0450f0f but kept `retroarch-nightly` (as it's not in scoop-extras).

**Comment by beyondmeat** [comment]
@rashil2000 not everyone needs or has extras bucket, which includes tons of stuff, so much stuff that github truncates how many apps are in that bucket.. Not very clean for people who use only use this bucket.

@Calinou

**Comment by rashil2000** [comment]
You can also create your own bucket if you want limited apps. Duplicating apps in official buckets is a bad idea and will confuse users.

**Pull request merged**